examples/test-upgrade/v100/privatelink-endpoint/v100/main.tf

```diff
@@ -0,0 +1,35 @@
+// This example is the same with the recent name of resource but will do nothing
```

Comment:
Better explanation, the why, but "this example is the same" may not make sense to someone not reading this in a PR (where the other example is right above it).   I'd fill that in.